types/culori/test/clamp.test.ts

```diff
@@ -0,0 +1,13 @@
+import { clampChroma, clampRgb, Color, GamutMode, Mode } from "culori/fn";
+
+declare const color: unknown;
+declare const mode: unknown;
+declare const rgbGamut: unknown;
+
+clampRgb(color as string); // $ExpectType Color | undefined
+clampRgb(color as Color); // $ExpectType Color
+
+clampChroma(color as string); // $ExpectType Color | undefined
+clampChroma(color as Color); // $ExpectType Color
+clampChroma(color as Color, mode as Mode); // $ExpectType Color
+clampChroma(color as Color, mode as Mode, rgbGamut as GamutMode); // $ExpectType Color
```

Comment:
This test doesn’t actually verify that the new type `GamutMode` is what you expect, since you never manifest a value of that type except through type assertion. It would be preferable to write realistic code for tests. You can take a call straight from the implementation test suite: `clampChroma('lch(80% 150 60)', 'lch', 'p3')`

Reply:
Alright, I've made the requested changes to the clamp test and it should be in my latest push. Thanks for helping me write better tests, Andrew!